styles/output/nursing-internal-pdf.css

```diff
@@ -1913,68 +1854,87 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   font-weight: bold;
   color: #C31427;
 }
-
 .os-table:not(.os-unstyled-container) .os-caption-container > .os-title {
   font-family: Mulish, sans-serif;
   font-weight: bold;
 }
-
 .os-table:not(.os-unstyled-container) .os-caption-container > .os-caption {
   font-family: IBM Plex Sans, sans-serif;
 }
-
-.os-eoc [data-type=problem] > div .os-table > table {
-  max-width: 100%;
-  table-layout: fixed;
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td {
+  border-top-color: #000000;
+  font-family: Mulish, sans-serif;
+  border-top-width: 0.1rem;
+  border-top-style: solid;
+  border-bottom-color: #000000;
+  border-bottom-width: 0.1rem;
+  border-bottom-style: solid;
+  padding-top: 0.7rem;
+  padding-bottom: 0.7rem;
+  font-weight: 700;
+  padding-left: 0.7rem;
+  padding-right: 0.7rem;
 }
-
-.os-eoc [data-type=problem] > div .os-table > table > tbody > tr > td:not(:only-of-type) {
-  text-align: center;
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td:not(:only-of-type) {
+  font-family: Mulish, sans-serif;
+  border-right: 0.025rem solid #000000;
   padding-top: 0.7rem;
   padding-bottom: 0.7rem;
-  padding-right: 4px;
-  padding-left: 4px;
+  padding-right: 0.7rem;
+  padding-left: 0.7rem;
+}
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td:not(:only-of-type) > strong {
+  word-break: break-word;
+}
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td[rowspan="2"]:not(:only-of-type):not(:first-child) {
+  border-left: 0.025rem solid #000000;
+}
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td:not(:only-of-type):not(:last-child) {
+  border-left: none;
+  border-right: 0.025rem solid #000000;
+}
+.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td:last-of-type > td:only-of-type {
+  border-right: none;
+}
+.os-table:not(.os-unstyled-container) > table > tbody > tr > td {
+  border-bottom: 0.025rem solid #000000;
+  padding: 0.7rem;
+}
+.os-table:not(.os-unstyled-container) > table col:not(:last-child) {
+  border-right: 0.025rem solid #000000;
 }
 
-.os-eoc [data-type=exercise-question] > div .os-table > table {
+.os-eoc [data-type=problem] > div .os-table > table, .os-eoc [data-type=exercise-question] > div .os-table > table {
   max-width: 100%;
   table-layout: fixed;
 }
-
-.os-eoc [data-type=exercise-question] > div .os-table > table > tbody > tr > td:not(:only-of-type) {
+.os-eoc [data-type=problem] > div .os-table > table > tbody > tr > td:not(:only-of-type), .os-eoc [data-type=exercise-question] > div .os-table > table > tbody > tr > td:not(:only-of-type) {
   text-align: center;
   padding-top: 0.7rem;
   padding-bottom: 0.7rem;
   padding-right: 4px;
   padding-left: 4px;
 }
 
-.os-eoc .os-problem-container > p + .os-table {
-  margin-top: 1rem;
-}
-
-.os-eob .os-solution-container > p + .os-table {
+.os-eoc .os-problem-container, .os-eob .os-solution-container {
```

Comment:
this is also not quite right -- missing the `> p + .os-table` part of the selector.

Reply:
This line is comes from the following mixin: 

```scss
@mixin table-top-spacing() {
   .os-eoc .os-problem-container, .os-eob .os-solution-container{
    margin-top: 1rem;
    }
 }
```
I've added the missing mixin for tables with exercises and now the proper selector is visible.

Reply:
Created a mixin for tables with exercises in general, and removed the table-top-spacing() mixin.